sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/AvroIO.java

```diff
@@ -326,6 +327,14 @@
       }
 
       @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        builder.add("schema", type);
```

Comment:
What happens here? `type` is a `Class`. Will it just get automatically converted to a `String`?

I would probably just drop the type. We aren't planning to specify the `Coder` for each transform as static metadata, are we?